src/Appwrite/Utopia/Response.php

```diff
@@ -342,10 +349,16 @@ public function __construct(SwooleHTTPResponse $response)
         parent::__construct($response);
     }
 
+    public function getSwoole(): SwooleHTTPResponse
+    {
+        return $this->swoole;
+    }
+
     /**
      * HTTP content types
      */
     public const CONTENT_TYPE_YAML = 'application/x-yaml';
+    public const CONTENT_TYPE_NULL = 'null';
```

Comment:
What is content type `null`? This doesn't seem HTTP compliant, how do we use it?

Reply:
This is used [here](https://github.com/appwrite/appwrite/blob/feat-graphql-support/src/Appwrite/Utopia/Response.php#L442) to allow the inner REST response to generate a payload, but not close the GraphQL request by sending the output